digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -137,6 +137,10 @@ public void ensureDatabaseCreated(String name)
             throws TDClientException
     {
         runWithRetry(() -> client.createDatabase(name), TDClientHttpConflictException.class);
+        // Check database existing and retry only once if it doesn't exit by session error(WM-763)
+        if (!client.existsDatabase(name)){
+            runWithRetry(() -> client.createDatabase(name), TDClientHttpConflictException.class);
```

Comment:
I confirmed to  you, createDatabase() will not fail even though already exist it.
So it is ok.